src/main/java/com/company/BookingList.java

```diff
@@ -0,0 +1,118 @@
+package com.company;
+import com.company.model.Client;
+import com.company.model.Movie;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.company.usinglist.MovieList.createFirstMovieList;
+import static com.company.usinglist.MovieList.createSecondMovieList;
+
+public class BookingList {
+    public static void AppRelise(){
+            List<Client> clients = new ArrayList<>();
+            try {
+                System.out.println("You must register!\n");
+
+                System.out.println("Enter your name : ");
```

Comment:
Why do you need to initialize so many Scanners? I think it should be a singleton object.